PhysicsTools/TensorFlow/src/NTSession.cc

```diff
@@ -507,6 +498,9 @@ Status NTSession::Run(const RunOptions& run_options,
     return errors::Cancelled("Run call was cancelled");
   }
 
+  args.runner = [this](Executor::Args::Closure c) {
+    SchedClosure(std::move(c));
+  };
   for (const auto& item : executors_and_keys->items) {
     item.executor->RunAsync(args, barrier->Get());
```

Comment:
this is the loop discussed in https://github.com/cms-sw/cmssw/pull/22260/files#r169140455

Reply:
✔︎